### PR TITLE
Removed Graphics, deprecated Canvas

### PR DESCRIPTION
**Pull request by kvark**
(this goes to v0.5 branch)
related to #737

### CODE REVIEW

File: src/extra/canvas.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub trait Window<R: Resources>: Output<R> {
     fn swap_buffers(&mut self);
 }
 
-/// A canvas with everything you need to draw on it.
+/// DEPRECATED
 pub struct Canvas<W, D: Device, F> {
```
Comment (csherratt) [inline src/extra/canvas.rs]: does `#[deprecated]` work?

Comment (kvark) [inline src/extra/canvas.rs]: I was under impression that this attribute is only reserved for standard libraries. Will try now.

Comment (kvark) [inline src/extra/canvas.rs]: `error: stability attributes may not be used outside of the standard library`

Comment (csherratt) [inline src/extra/canvas.rs]: :( oh well.

**Comment by csherratt** [comment]
lgtm

**Pull request merged**